backend/src/services/PropertyService.ts

```diff
@@ -0,0 +1,14 @@
+import { Types } from 'mongoose';
+import Property, { IProperty } from '../model/Property';
+
+export const createProperty = (postalCode: string, streetName: string, streetNumber: number, landlordId: Types.ObjectId): Promise<IProperty> => {
+  return new Promise((resolve, reject) => {
+    Property.create({ postalCode, streetName, streetNumber, landlordId })
+      .then((result) => {
+        resolve(result);
+      })
+      .catch((error: Error) => {
+        reject(new Error(`Failed to create property: ${error}`));
+      });
+  });
+};
```

Comment:
Since `Property.create` already returns a promise does a new promise have to be created? Something like
```suggestion
export const createProperty = (postalCode: string, streetName: string, streetNumber: number, landlordId: Types.ObjectId): Promise<IProperty> => {
    return Property.create({ postalCode, streetName, streetNumber, landlordId })
};
```